packages/components/src/List/Toolbar/ColumnChooserButton/ColumnChooserButton.component.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Overlay, Popover } from 'react-bootstrap';
+import ColumnChooser from './ColumnChooser';
+import getDefaultT from '../../../translate';
+import ActionButton from '../../../Actions/ActionButton';
+
+export default function ColumnChooserButton({
+	children,
+	columns,
+	filterValue,
+	id,
+	initialOpenedPopover,
+	placement = 'left',
+	nbLockedLeftItems,
+	submit,
+	t = getDefaultT(),
+}) {
+	const [opened, setOpened] = useState(initialOpenedPopover || false);
+	const [buttonRef, setButtonRef] = useState(null);
+	const changeOpened = () => setOpened(!opened);
+	const closePopover = () => setOpened(false);
+	const colChooserButtonId = `${id}-column-chooser`;
```

Comment:
This is not useful IMO. User pass an id to the chooser, you don't need to add another context.
But in Toolbar component, add this context before passing the id to column chooser, like the other elements in the toolbar.

Reply:
Don't agree but I change it.